syntax = "proto3";


message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply);
}

message AllTemplatesRequest{
}

message AllTemplatesReply{
    repeated string names = 1;
}

service AllTemplatesService{
  rpc GetAllTemplates (AllTemplatesRequest) returns (AllTemplatesReply);
}